python/paddle/v2/reader/creator.py

```diff
@@ -57,22 +57,51 @@ def reader():
     return reader
 
 
-def recordio(path):
+def recordio_local(paths):
     """
-    Creates a data reader that outputs record one one by one from given recordio file
-    :path: path of recordio file
-    :returns: data reader of recordio file
+    Creates a data reader that outputs record one one by one 
+        from given local recordio fils path.
```

Comment:
Maybe change `Creates a data reader that outputs record one one by one from given local recordio fils path.` to `Creates a data reader from given RecordIO file paths separated by ",", glob pattern is supported.`